lib/node_modules/@stdlib/math/base/special/cinv/src/main.c

```diff
@@ -0,0 +1,99 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/cinv.h"
+#include "stdlib/math/base/special/abs.h"
+#include "stdlib/constants/float64/max.h"
+#include "stdlib/constants/float64/eps.h"
+#include "stdlib/constants/float64/smallest_normal.h"
+#include "stdlib/complex/float64.h"
+#include "stdlib/complex/reim.h"
+
+
+// VARIABLES //
+
+const double LARGE_THRESHOLD = STDLIB_CONSTANT_FLOAT64_MAX * 0.5;
```

Comment:
I missed this here and possibly in other PRs involving constants, but we need to make these constants `static`; otherwise, they are visible outside the translation unit.